src/decisionengine/framework/modules/Module.py

```diff
@@ -21,6 +28,9 @@ def get_data_block(self):
     def set_data_block(self, data_block):
         self.data_block = data_block
 
+        #def get_logger(self):
+#return self.logger
+
```

Comment:
Can probably drop these three debug lines

Reply:
Thanks. I did mean to do that, and now I have.